CsCore/xUnitTests/src/Plugins/CsCoreXUnitTests/com/csutil/tests/http/apis/OpenAiTests.cs

```diff
@@ -199,6 +201,76 @@ public enum Emotion { happy, sad, angry }
 
         }
 
+
+        [Fact]
+        public async Task ExampleTTSandSTT() {
+            string textToTest = "hello world";
+            Stream TTSResponseStream = await TTS(new Audio.TTSRequest() { input = textToTest });
+            Assert.NotNull(TTSResponseStream);
+
+            Audio.STTResponse STTResponse = await STT(new Audio.STTRequest() { file = TTSResponseStream });
+            Assert.NotEmpty(STTResponse.text);
+            Log.d(STTResponse.text);
+            //fails when textToTest contains numbers
+            Assert.Equal(formatString(STTResponse.text), formatString(textToTest));
+        }
+
+        private string formatString(string str) {
+            //helper function to format TTS and STT converted texts to test weather input text and output text is equal
+            return new String(str.ToCharArray().Where(c => !Char.IsWhiteSpace(c) && Char.IsLetterOrDigit(c))
+            .ToArray()).ToLower();
+        }
+
+        public async Task<Stream> TTS(Audio.TTSRequest requestParam) {
```

Comment:
This method here we should move into the OpenAi class, since it does the actual rest request and can then also make use of the openai key that the openai class already has as a state

Reply:
TextToSpeech(previously TTS) and SpeechToText(previously STT) methods have been moved to OpenAi class